pages/common/lp.md

```diff
@@ -0,0 +1,31 @@
+# lp
+
+> print files
+
+- List printers present on the machine.
+
+`lpstat -p | awk '{print $2}`
+
+- List the default printer.
+
+`lpstat -d`
+
+- Print the output of a command.
+
+`echo "test" | lp -d {{printer_name}}`
+
+- Print to the default printer.
+
+`lpstat {{filename}}`
+
+- Print a file to a named printer. 
+
+`lp -d {{printer_name}} {{path/to/filename}}`
+
+- Print N copies to default printer (example used is 2).
```

Comment:
"N copies of a file" (for consistency with above examples). Also, the 2 in the example should be `{{N}}`.